docs/cloud/faq/README.md

```diff
@@ -120,15 +120,9 @@ Please ensure to learn about the SLA levels provided by Event Store Cloud in our
 
 The Cloud Console access audit log is in our roadmap, we will present the details of the schedule on the public [roadmap](https://www.eventstore.com/event-store-cloud) page.
 
-#### Does Event Store have a security policy?
```

Comment:
I think that this question is still valid. People were asking for that. It might be worth asking Andres if we have something to show in public or a dedicated point of contact to get it.

Reply:
going to re-add it & we can wait for further info from the CISO .
( we need to get the ISO / SOC out )